src/MIDebugEngine/Engine.Impl/EngineUtils.cs

```diff
@@ -23,14 +23,14 @@ internal static string AsAddr(ulong addr, bool is64bit)
 
         internal static string GetAddressDescription(DebuggedProcess proc, ulong ip)
         {
-            string description = null;
-            proc.WorkerThread.RunOperation(async () =>
+            Task<string> description = null;
+            proc.WorkerThread.RunOperation(() =>
             {
-                description = await EngineUtils.GetAddressDescriptionAsync(proc, ip);
+                description = EngineUtils.GetAddressDescriptionAsync(proc, ip);
             }
             );
 
-            return description;
+            return String.IsNullOrEmpty(description.Result) ? "HORSE" : description.Result;
```

Comment:
`HORSE` is an address?

Reply:
No. it was a test phrase that i forgot. i'll remove